0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -26,70 +28,31 @@ This document does **not** specify how payment requests are communicated from a
 
 Interledger Payment Requests are intended to be used in higher-level protocols as follows:
 
-1. Recipient creates the payment request and generates the condition using one of algorithms outlined in [Condition Generation](#condition-generation).
-2. Recipient communicates the payment request to the sender.
-3. Sender generates an ILP Packet from the payment request.
-4. Sender prepares a local ledger transfer to a connector, held pending the execution condition given in the payment request and with the ILP Packet attached.
-5. Connectors forward the payment until it reaches the recipient.
-6. Recipient receives a notification of a held transfer with an ILP Packet attached.
-7. Recipient checks the transfer amount against the ILP Packet and checks the payment request has not expired.
-8. Recipient regenerates the condition and fulfillment from the ILP Packet attached to the incoming transfer, using the same algorithm as before, and fulfills the transfer's condition. If the condition generated does not match the one in the transfer it means that the ILP Packet has been tampered with and the transfer goes unfulfilled.
+1. Recipient creates the ILP Packet and generates the condition using one of algorithms outlined in [Condition Generation](#condition-generation).
+2. Recipient communicates the ILP Packet and condition to the sender.
+3. Sender prepares a local ledger transfer to a connector, held pending the execution condition provided by the recipient and with the ILP Packet provided by the recipient attached.
```

Comment:
Would be great to link to an explanation of "connectors" in the Architecture RFC or somewhere. The term doesn't make much sense if you're not familiar with Interledger.